### PR TITLE
privnet: add Bridge contract to genesis config

### PR DESCRIPTION
**Pull request by AnnaShaleva**
The Bridge config is exactly the same that we use for NeoX T2. Also, make 7-nodes privnet config closer to T2 config.

### CODE REVIEW

**Comment by AnnaShaleva** [comment]
@roman-khimov, I'm not sure it's a good idea to keep "unknown" version of contract in the privnet genesis, but at the same time it's still a way to check that Testnet environment works, so I think we need to merge this PR. What do you think?

File: privnet/seven/genesis_privnet.json [hunk 1]
```diff
@@ -25,18 +25,36 @@
...
-      "coinbase": "0x1212100000000000000000000000000000000001"
+      "coinbase": "0x1212100000000000000000000000000000000003"
     }
   },
   "difficulty": "1",
-  "gasLimit": "7000000",
+  "gasLimit": "30000000",
   "alloc": {
-    "74f4effb0b538baec703346b03b6d9292f53a4cd": { "balance": "100000000000000000000" },
+    "74f4effb0b538baec703346b03b6d9292f53a4cd": { "balance": "100000000000000000000000" },
```
Comment (roman-khimov) [inline privnet/seven/genesis_privnet.json:34]: Why this one?

**Review by roman-khimov — COMMENTED** [review]
We can have it as is for now (synchronized with T2) and then update when needed.

File: privnet/seven/genesis_privnet.json (hunk 1 above)
Comment (AnnaShaleva) [reply privnet/seven/genesis_privnet.json:34]: Just a single validator from the batch to follow the notion of T2 config.

**Comment by AnnaShaleva** [comment]
> We can have it as is for now (synchronized with T2)

Good, then current version is in sync.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**